doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -2441,8 +2496,39 @@ intermediate statement.
 The have tactic.
 ````````````````
 
-.. tacn:: have : @term
-   :name: have
+.. tacn:: have @ssrhpats_wtransp {* @ssrbinder } @ssrhavefwd
+   :name: have (ssreflect)
+
+   .. insertprodn ssrhpats_wtransp ssrhavefwd
+
+   .. prodn::
+      ssrhpats_wtransp ::= {? @ssripats }
+      | {? @ssripats } @ {? @ssripats }
+      ssripats ::= {+ @i_item }
+      s_item ::= //=
+      | /=
+      | / @natural {| / {? @natural = } | = | /= }
+      | // {? @natural = }
+      | //
```

Comment:
The variants `/n=` `/n/` and `//n=` are parsed but never implemented, see #9195 
The best option would be to remove them/comment them in the parser as well.

Reply:
Gone from the doc.  I'll open another PR for SSR mlg fixups.